app/notification_push_test.go

```diff
@@ -898,7 +898,7 @@ func TestGetPushNotificationMessage(t *testing.T) {
 	}
 }
 
-func TestBuildPushNotificationMessage(t *testing.T) {
+func TestBuildPushNotificationMessageMentions(t *testing.T) {
```

Comment:
I miss in this test some cases testing the wrong cases, for example, invalid user or channel.